core/src/main/resources/db/migration/V28__add_campaign_created_date.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE `Campaign`
+ADD COLUMN created_at DATETIME(3) NOT NULL DEFAULT NOW();
+
+ALTER TABLE `Package`
+ADD COLUMN created_at_temp DATETIME(3) NOT NULL DEFAULT NOW();
+
+UPDATE `Package`
+SET created_at_temp = createdAt;
+
+ALTER TABLE `Package`
+DROP COLUMN createdAt;
+ALTER TABLE `Package`
+CHANGE created_at_temp created_at DATETIME(3) NOT NULL DEFAULT NOW();
```

Comment:
Let's leave this like this for now, but just so we do it better next time:

We should use temporary tables when doing stuff like this:

1. Create temporary table with the package_uuid, created_at tuples (see CREATE TEMPORARY TABLE)
2. drop created_at column
3. add created_at column with DATETIME(3)
4. update created_at column with values from temporary table

Using a temporary table saves us from adding a new column to the database, with can be an expensive operation depending on the size of the table.